SimTracker/TrackTriggerAssociation/plugins/TTTrackAssociator.h

```diff
@@ -79,6 +78,13 @@ TTTrackAssociator<T>::TTTrackAssociator(const edm::ParameterSet& iConfig) {
   TTTracksInputTags = iConfig.getParameter<std::vector<edm::InputTag> >("TTTracks");
   TTClusterTruthToken = consumes<TTClusterAssociationMap<T> >(iConfig.getParameter<edm::InputTag>("TTClusterTruth"));
   TTStubTruthToken = consumes<TTStubAssociationMap<T> >(iConfig.getParameter<edm::InputTag>("TTStubTruth"));
+  TTTrackAllowOneFalse2SStub = iConfig.getParameter<bool>("TTTrackAllowOneFalse2SStub");
+  std::cout << "TTTrackAssociator: ";
```

Comment:
cout is not allowed, remove or use MessageLogger

Reply:
fixed